packages/react-components/src/DocsComponents/FluentDocsHeader.stories.tsx

```diff
@@ -12,7 +14,11 @@ const useStyles = makeStyles({
 /**
  * Sticky header over the entire docs page
  */
-export const FluentDocsHeader = () => {
+export const FluentDocsHeader: React.FC<{ storybookGlobals: FluentGlobals }> = ({ storybookGlobals }) => {
   const styles = useStyles();
-  return <div className={styles.root}>{/** TODO add theme+version picker */}</div>;
+  return (
+    <div className={styles.root}>
+      <ThemePicker selectedThemeId={storybookGlobals['storybook/fluentui-react-addon/theme']} />
```

Comment:
can we export this `'storybook/fluentui-react-addon/theme'` so its part of public API of the package - thus prevent breaking changes if it would be changed ?

Reply:
yeah `THEM_ID` is already exported,  I just overlooked it here